Coinjoin confirmation page redesign

komret opened the pull request:
## Description

Simplification of coinjoin confirmation page:
- removal of pre-defined strategies
- removal of fee calculation
- redesign

Follow-ups:
- custom setup #7431
- remnoval of fast fee rates median #7432

## Related Issue

Resolve #7163

## Screenshots:
![Screenshot 2023-01-20 at 17 50 57](https://user-images.githubusercontent.com/42465546/213765068-2fdae976-2b70-4775-8c4d-53977c936eb5.png)

matejkriz commented on packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx:
```diff
@@ -112,123 +146,82 @@ export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProp
+                <Tiles>
+                    {tiles.map(tile => (
+                        <Tile
+                            key={tile.description.props.title}
```
isn't that undefined?
<img width="522" alt="image" src="https://user-images.githubusercontent.com/3729633/214810652-46220385-1887-4fce-81c7-f99274ed36d5.png">

komret replied:
5eca8e7fc665902d1c5124883fa1cefb2034f08f

matejkriz commented:
[packages/Trezor_T_Top_Black_Connected.png](https://github.com/trezor/trezor-suite/pull/7433#diff-7dd5971d841e8c5d1c092cc76b74bbe4b0828ea2305319d77d536e6c1196339e) image is pushed by mistake, right? @komret 

And the quality of [packages/suite-data/files/images/png/Trezor_T_Connected.png](https://github.com/trezor/trezor-suite/pull/7433#diff-9967f54cadde1e5617f89f49a758d1ad32b7f33ce415108054222edf28a3157c) is too bad, would be nice have sligthly better quality and provide also @2x version

matejkriz commented on packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx:
```diff
@@ -248,13 +241,11 @@ export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProp
-            </StyledTooltipButton>
+            <Tooltip content={getButtonTooltipMessage()}>
+                <StyledButton onClick={anonymize} isDisabled={isDisabled}>
+                    <Translation id="TR_ANONYMIZE" />
```
By the design, there is a new copy different from the one in BalanceSection ("Start coinjoin" vs "Anonymize") so maybe it makes sense to create a new message id here.

komret replied:
5eca8e7fc665902d1c5124883fa1cefb2034f08f

matejkriz commented:
Except the Tile key, the image of model T and notes for the design/copy,  it looks good to me 👍

komret commented:
> [packages/Trezor_T_Top_Black_Connected.png](https://github.com/trezor/trezor-suite/pull/7433#diff-7dd5971d841e8c5d1c092cc76b74bbe4b0828ea2305319d77d536e6c1196339e) image is pushed by mistake, right? @komret

That's right, removed in 5eca8e7fc665902d1c5124883fa1cefb2034f08f.

> And the quality of [packages/suite-data/files/images/png/Trezor_T_Connected.png](https://github.com/trezor/trezor-suite/pull/7433#diff-9967f54cadde1e5617f89f49a758d1ad32b7f33ce415108054222edf28a3157c) is too bad, would be nice have sligthly better quality and provide also @2x version

Also fixed in 5eca8e7fc665902d1c5124883fa1cefb2034f08f.

matejkriz reviewed (approved):
nice, please rebase&merge
